### ISSUE TITLE
metadata

### ISSUE THREAD

**Issue by ghost** [download13/ipfstube#17]
should be able to support metadata, for example images via `og:image`

http://ogp.me

**Comment by download13** [comment]
The player is just a static page that loads the requested video using JS.

What did you imagine as the role of the graph links for ipfstube?

**Comment by ghost** [comment]
@download13 if a person wants to link to an ipfstube page, `og:image` is needed else no thumbnail will be possible

**Comment by download13** [comment]
I don't really have a way to extract thumbnails from the videos. Do you mean just for the site logo?

**Comment by ghost** [comment]
no, for the actual videos - if you dont have way to extract - perhaps option could be added for user to upload thumbnail along with video
